api-testing/multicloud-api/src/main/java/com/soda/HttpHandler.java

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
How are we deciding on package names?
com.soda?
Can't we make this generic in terms of soda foundation.
1 more point for the project structure. 
Please place the code in specific packages according to their functionality For example: Utility and Http Handlers can be seggregated.

Reply:
Please check this comment [Link](https://github.com/sodafoundation/qa/pull/2#issuecomment-637337884)

Reply:
What to check?

Reply:
This is repeated question. Already discussed with @anvithks

Reply:
https://github.com/Click2cloud-Wadhai @himanshuvar earlier this path name was */com/opensds/* so requested to change it to */com/soda/* ..Any better suggestion?

Reply:
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html
Companies use their reversed Internet domain name to begin their package names—for example, com.example.mypackage for a package named mypackage created by a programmer at example.com.

Name collisions that occur within a single company need to be handled by convention within that company, perhaps by including the region or the project name after the company name (for example, com.example.region.mypackage).

I  suggest use io.sodafoundation.<package_name>

Reply:
And please use packages purposefully.
I mean place all the classes into the package which they belong to.

Reply:
I have renamed package name and move this class in separate package. Please review.